comet-java-client/src/main/java/ml/comet/experiment/OnlineExperimentImpl.java

```diff
@@ -261,15 +277,21 @@ public String getExperimentKey() {
         return Optional.ofNullable(this.experimentLink);
     }
 
+    @Override
+    public void logMetric(String metricName, Object metricValue, Long step) {
```

Comment:
change 
```
Long step
```
to 
```
long step
```
or change all other epoch and step method parameters to Long

Reply:
Nice catch! Updated to primitive.